tests/test_filters.py

```diff
@@ -388,6 +388,12 @@ def test_conjoined_true(self):
         f = MultipleChoiceFilter(conjoined=True)
         self.assertTrue(f.conjoined)
 
+    def test_is_noop_false(self):
+        f = MultipleChoiceFilter(required=False)
+        f.always_filter = False
+        qs = mock.Mock(spec='filter')
+        self.assertFalse(f.is_noop(qs, ['value']))
```

Comment:
I'm unsure why `is_noop` does not just get `None, None` as input here — the method body does not seem to use `qs, ['value']`. Am I missing something?

Reply:
`is_noop` does use the value

https://github.com/carltongibson/django-filter/blob/a5b4d0299ec04dcb103dc37291001fb268107c66/django_filters/filters.py#L226

I think the `qs` is provided because someone may want to override the behavior of `is_noop`, and having access to the `qs` is likely useful.